content/logs/log_collection/docker.md

```diff
@@ -125,7 +125,7 @@ For more examples of configuration files or Agent capabilities (such as filterin
 
 Since version 6.2 of the Datadog Agent, you can configure log collection directly in the container labels.
 
-Autodiscovery expects labels to look like these format, depending on the file type:
+Autodiscovery expects labels to look like this format, depending on the file type:
```

Comment:
Autodiscovery expects labels to follow this format, depending on the file type:

Reply:
@cswatt  Thanks for the comments!
Please feel free to make any further changes that are needed.